sdk/rust/src/http/router.rs

```diff
@@ -19,6 +19,18 @@ impl Default for Router {
     }
 }
 
+impl Display for Router {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str("Registered routes:\n")?;
+        for (method, router) in &self.methods_map {
+            router.iter().for_each(|route| {
+                f.write_fmt(format_args!("- {}: {}\n", method, route.0)).unwrap();
```

Comment:
Could we use the `write!` macro here?

Reply:
Sorry meant to hit reply! 

Similarly,  there is a `writeln!` if you prefer not having the `\n` in the format string.

Reply:
so obvious 🤦🏼‍♂️